public/js/app/pages/CoursePage.jsx

```diff
@@ -163,6 +163,8 @@ function CoursePage() {
             </div>
           )}
 
+          <InfoModal title="Test" infoText="<b>yea test</b>" type="html" closeLabel="Close" ariaLabel="SOme aria" />
```

Comment:
_yea test_ - this wasn't supposed to be included?

Reply:
Fixed 0098888